libtenzir/include/tenzir/defaults.hpp

```diff
@@ -163,27 +163,30 @@ inline constexpr const size_t rotate_files = 3;
 
 } // namespace logger
 
-// -- constants for the various builtin REST endpoints -------------------------
-namespace rest {
+// -- constants for the builtin REST endpoints -------------------------
+namespace api {
 
-/// Settings related to the `/query` endpoint.
-namespace query {
+/// The maximum response size for one request.
+inline constexpr size_t max_response_size = size_t{124} * 1'024; // 124 KiB
 
-/// The maximum number of results
-inline constexpr const size_t limit = 50;
+namespace serve {
 
-/// Whether to flatten nested fields.
-inline constexpr const bool flatten = false;
+/// The duration for which old results are kept after being fetched.
```

Comment:
This comment is incorrect. The timeout only applies for the last result set for which the returned continuation token is null. For all other result sets we keep the results around until the next continuation token is used in a request successfully.